### DISCUSSION TITLE
Skypack shows Alpine's German README

### DISCUSSION THREAD

**Discussion by earthboundkid** [alpinejs/alpine#1407] [7. Bugs (archived)]
Background: Skypack is an unpkg-like CDN from the Snowpack/Pika people.

For some reason, the entry for Alpine shows the German README instead of English: https://www.skypack.dev/view/alpinejs. Maybe it's getting bad info off of package.json or something on NPM?

**Comment by KevinBatdorf** [comment]
Looks like they just grab the first one, which happens to be german.

**Comment by calebporzio** [comment]
Ew, any ideas on the best way to fix this? do we have to add some sort of prefix that's after "R" to all the other languages?

**Comment by earthboundkid** [comment]
My suspicion is that it will have to be fixed on their side. :-/

**Comment by drwpow** [comment]
👋🏻  Gave a longer explanation here: https://github.com/snowpackjs/skypack-cdn/issues/68, but the short version is that this is the data we get directly from npm (see the `"readme"` and `"readmeFilename"` keys here: http://registry.npmjs.com/alpinejs).

> Looks like they just grab the first one, which happens to be german.

I can‘t confirm it, but I agree that this seems to be the case. Of all the `README*` files in this repo, `README.de.md` comes first, alphabetically

> Ew, any ideas on the best way to fix this? do we have to add some sort of prefix that's after "R" to all the other languages?

I have no idea why, but https://www.npmjs.com/package/alpinejs shows English 🤷🏻 . We can fix it on Skypack’s end for all packages, but if you care about any other CDNs (e.g. [Yarn](https://yarnpkg.com/package/alpinejs)) or other things talking to npm’s API, then yeah perhaps moving the non-English files into a `docs/` or `lang/` (or whatever) folder would resolve the issue in npm‘s API data.

This is the first time I’ve seen this so I wouldn‘t have predicted it either 😄

**Comment by HugoDF** [comment]
> I have no idea why, but https://www.npmjs.com/package/alpinejs shows English 🤷🏻

that's probably because it explicitly reads the README.md

**Comment by KevinBatdorf** [comment]
For what it's worth, I've had only positive experiences with NPM support so I bet they would be helpful here.

**Comment by SimoTod** [comment]
We could also npm-ignore those files so they don't get uploaded.

**Comment by calebporzio** [comment]
Oooh if npm ignore works, that's the cleanest.

**Comment by mattrossman** [comment]
Now it shows the Arabic version on Skypack, this threw me off as a first timer 😅

**Comment by FredKSchott** [comment]
Hey all, Fred from Skypack here. For some reason, npm's registry sends back different languages for alpinejs: http://registry.npmjs.org/alpinejs you can see Arabic in the "readme" section of the JSON object. 

I'll look into adding a one-off fix for our search catalog

**Comment by SimoTod** [comment]
@FredKSchott I believe it's a bug in npm to be fair. It seams like they send the first file they can find in the root starting with the word `README`. :)

**Comment by HugoDF** [comment]
Seems like the solution is this one (npmignore the non English READMEs) @FredKSchott https://github.com/alpinejs/alpine/issues/867#issuecomment-736638651
